### ISSUE TITLE
Updating NDVI Threshold

### ISSUE THREAD

**Issue by cjthomas730** [SkyTruth/MTR#56]
_Blocks #49_

Based upon Andrew's research (see #52), we want to create a threshold for each year of the study. To do this we need to:
- [x] implement the script written by Andrew which utilizes the Otsu method
- [x] select ~30 training sites and an appropriate buffer distance to use for the Otsu method, use the average or median NDVI of the training sites as threshold for that year
   - possibly select sites by using App. Voices' Permit Data

**Comment by apericak** [comment]
@TwigBeard I've updated the Otsu script (and @cjthomas730 just committed it to Master). Within the script I included the step-by-step instructions for running it, but of course let me know if you have any issues/questions. The link to the spreadsheet for recording the data is in the code, but you can also [click here](https://docs.google.com/spreadsheets/d/1WpR4VJkzjAqsJ6UzWeTVM80JRJBS06x-s-x4kDZd6Kc/edit?usp=sharing). In terms of knowing who's doing what, you can start at the top of the list and work your way down; I'll start at the bottom and work up. Check the box once you've completed it.

- [ ] 1984
- [ ] 1985
- [ ] 1986
- [ ] 1987
- [ ] 1988
- [ ] 1989
- [ ] 1990
- [ ] 1991
- [ ] 1992
- [ ] 1993
- [ ] 1994
- [ ] 1995
- [ ] 1996
- [ ] 1997
- [ ] 1998
- [ ] 1999
- [ ] 2000
- [ ] 2001
- [ ] 2002
- [ ] 2003
- [ ] 2004
- [ ] 2005
- [ ] 2006
- [ ] 2007
- [ ] 2008
- [ ] 2009
- [ ] 2010
- [ ] 2011
- [ ] 2012
- [ ] 2013
- [ ] 2014
- [ ] 2015

**Comment by cjthomas730** [comment]
@apericak @TwigBeard do you need more help with this? If yes, let me know and @JerrilynGoldberg and I can help out with a few years

**Comment by apericak** [comment]
@cjthomas730 @JerrilynGoldberg Yah you all can definitely help--as I mentioned, the instructions are located right on the script, but if you all want I can call in and demo it for you via screenshare (I did that for @TwigBeard a few days ago). 

I ran a few of these this years this weekend and it went really fast, but now that we're back to the workweek the processing steps are much slower; a really good example of how GEE uses excess Google capacity and how apparently there's so much more excess over the weekends. That being said, I think having more people working on this would therefore mean we get done faster. 

Since you all are in the same office, it's probably easiest to start from the top of the list, working down, and coordinate with each other about what years you are each doing. I'll continue working from the bottom up.
